optimizely/project_config.py

```diff
@@ -85,18 +88,24 @@ def __init__(self, datafile, logger, error_handler):
         for layer in self.rollout_id_map.values():
             for experiment in layer.experiments:
                 self.experiment_key_map[experiment['key']] = entities.Experiment(**experiment)
+                self.experimentID_map[experiment['id']] = entities.Experiment(**experiment)
 
         self.audience_id_map = self._deserialize_audience(self.audience_id_map)
         for group in self.group_id_map.values():
             experiments_in_group_key_map = self._generate_key_map(group.experiments, 'key', entities.Experiment)
             for experiment in experiments_in_group_key_map.values():
                 experiment.__dict__.update({'groupId': group.id, 'groupPolicy': group.policy})
             self.experiment_key_map.update(experiments_in_group_key_map)
+            experiments_in_group_id_map = self._generate_key_map(group.experiments, 'id', entities.Experiment)
```

Comment:
To avoid confusion maybe we should rename this function or create a new one for ids.